src/cmd/turnkey/pkg/activities.go

```diff
@@ -32,47 +32,46 @@ var activitiesListCmd = &cobra.Command{
 	Use:   "list",
 	Short: "Return the set of activities for an organization",
 	Run: func(cmd *cobra.Command, args []string) {
-		activitiesFilter := make([]models.V1ActivityStatus, len(activitiesListStatus))
+		activitiesFilter := make([]models.ActivityStatus, len(activitiesListStatus))
 
 		for i, s := range activitiesListStatus {
 			if s == Help {
-				Output(models.ActivityStatuses())
+				Output(models.ActivityStatusEnum)
```

Comment:
I don't love the name `XEnum` to represent a slice... but I didn't want to heavily modify the template. LMK if this is too bothersome